xla/service/gpu/transforms/cudnn_fused_mha_rewriter_test.cc

```diff
@@ -83,6 +84,12 @@ class CudnnFusedMhaRewriterTestHloTest : public HloTestBase {
     return se::dnn::VersionInfo(9, 0, 0);
   }
 
+  se::dnn::VersionInfo GetRealCudnnVersion() {
```

Comment:
Sorry that I missed this before. This test is not actually running stuff on real hardware, so we should not be using the real cudnn version, but instead test for a specific cudnn version that we hardcode. Can you please create a new test instead of adjusting an existing test, and in the new test pass a supported cudnn version and hopper as cc, so that the new code is triggered?

Reply:
sure. I can add a separate test. And can you remind me how the test executed without actual hardware?

Reply:
> sure. I can add a separate test. And can you remind me how the test executed without actual hardware?

You can just pass any CudaComputeCapability that you want, e.g.

```se::CudaComputeCapability(9, 0);```

and

```se::dnn::VersionInfo(9, 5, 0)```

The test is just running the rewriter pass and then pattern matching the changed HLO. No need for any real hardware for that.

Reply:
Thx for clarification. New changes are pushed. PTAL.